frontend/appflowy_flutter/packages/flowy_infra/lib/utils/color_converter.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+class ColorConverter implements JsonConverter<Color, String> {
+  const ColorConverter();
+
+  static const Color fallback = Colors.transparent;
```

Comment:
If the `FlowyColorScheme` API changes. The parser will provide `transparent` as the default value. 

Another way to fail silently is to pass over dynamic themes that can't be parsed. I.e., they won't be included if the app is updated since the theme is invalid. I thought this would be the best way to fail silently.